tools/experimental/dynamic_shapes/torchfuzz/multi_process_fuzzer.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
+"""
+Multi-process fuzzer library that uses worker processes to execute fuzzer.py with different seeds.
+"""
+
+import multiprocessing as mp
+import re
+import subprocess
+import sys
+import time
+
+
+try:
+    from tqdm import tqdm
+
+    HAS_TQDM = True
+except ImportError:
+    HAS_TQDM = False
+
+
+def persist_print(msg):
+    """Print messages that persist with tqdm progress bars."""
+    try:
+        if HAS_TQDM:
+            # Keep prints on the same stream as the bar
+            tqdm.write(msg, file=sys.stderr)
+        else:
+            print(msg, file=sys.stderr, flush=True)
+    except BrokenPipeError:
+        import os
+
+        os.makedirs("/tmp/torchfuzz", exist_ok=True)
+        with open("/tmp/torchfuzz/crash.log", "a") as f:
+            f.write(f"BrokenPipeError: {msg}\n")
+
+
+# List of regex patterns for ignore bucket
+IGNORE_PATTERNS: list[re.Pattern] = [
```

Comment:
why are we ignoring
is not define
was not declared in this scope
are those bugs in the fuzzer or bugs found in the fuzzer

also its nice to keep those ad dict so we track know there is a gh issue associated with each